src/main/java/org/folio/innreach/domain/processor/ItemShippedInnReachCirculationProcessor.java

```diff
@@ -0,0 +1,67 @@
+package org.folio.innreach.domain.processor;
+
+import java.util.Objects;
+import java.util.UUID;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+import org.folio.innreach.client.InventoryClient;
+import org.folio.innreach.domain.CirculationOperation;
+import org.folio.innreach.domain.entity.InnReachTransaction;
+import org.folio.innreach.domain.entity.TransactionPatronHold;
+import org.folio.innreach.domain.exception.EntityNotFoundException;
+import org.folio.innreach.dto.InnReachResponseDTO;
+import org.folio.innreach.dto.TransactionHoldDTO;
+import org.folio.innreach.repository.InnReachTransactionRepository;
+
+@Log4j2
+@Service
+@RequiredArgsConstructor
+public class ItemShippedInnReachCirculationProcessor implements InnReachCirculationProcessor {
+
+  private final InnReachTransactionRepository transactionRepository;
+  private final InventoryClient inventoryClient;
+
+  @Override
+  public boolean canProcess(String circulationOperationName) {
+    return CirculationOperation.ITEM_SHIPPED.getOperationName().equals(circulationOperationName);
+  }
+
+  @Override
+  @Transactional
+  public InnReachResponseDTO process(String trackingId, String centralCode, TransactionHoldDTO transactionHold) {
+    var innReachTransaction = transactionRepository.findByTrackingIdAndCentralServerCode(trackingId, centralCode)
+        .orElseThrow(() -> new EntityNotFoundException(String.format(
+            "InnReach transaction with trackingId [%s] and centralCode [%s] not found", trackingId, centralCode)));
+
+    var itemBarcode = transactionHold.getShippedItemBarcode();
```

Comment:
![image](https://user-images.githubusercontent.com/42244720/140068965-82644edf-fe3d-4980-afdc-ec4ffb7c2d66.png)
the request has "itemBarcode" property, from the other side TransactionHoldDTO contains field (and JsonProperty) named "shippedItemBarcode"..
How is the property from json request mapped to the class property?

Reply:
@dmtkachenko, I've changed the payloa, now it's CircualtionRequestDTO that contains the itemBarcode